OrrRock/OrrRock/ViewControllers/VideoCollectionViewController/DateAndGymEditViewController.swift

```diff
@@ -0,0 +1,245 @@
+//
+//  DateAndGymEditViewController.swift
+//  OrrRock
+//
+//  Created by dohankim on 2022/10/25.
+//
+
+import UIKit
+import SnapKit
+
+class DateAndGymEditViewController: UIViewController , UISheetPresentationControllerDelegate{
+
+    
+    override var sheetPresentationController: UISheetPresentationController {
+        presentationController as! UISheetPresentationController
+    }
+    
+    // MARK: gym view compenents
+    let gymContentView : UIView = {
+        let view = UIView()
+        view.alpha = 0.0
+        return view
+    }()
+    
+    let gymNameLabel : UILabel = {
+        let label = UILabel()
+        label.text = "해당 암장의 이름을 적어주세요"
+        label.font = UIFont.boldSystemFont(ofSize: 17)
+        label.textColor = .orrBlack
+        label.backgroundColor = .orrGray1
+        return label
+    }()
+
+    let gymTextField : UnderlinedTextField = {
```

Comment:
p1;

```suggestion
    let gymContentView: UIView = {
        let view = UIView()
        view.alpha = 0.0
        return view
    }()
    
    let gymNameLabel: UILabel = {
        let label = UILabel()
        label.text = "해당 암장의 이름을 적어주세요"
        label.font = UIFont.boldSystemFont(ofSize: 17)
        label.textColor = .orrBlack
        label.backgroundColor = .orrGray1
        return label
    }()

    let gymTextField: UnderlinedTextField = {
```

<img width="870" alt="image" src="https://user-images.githubusercontent.com/78950704/197631648-fae8183c-c383-4b01-a01a-a53905e87ea5.png">

콜론(:)을 쓸 때에는 콜론의 오른쪽에만 공백을 둬요!
반영 부탁드립니다!

Reply:
이전 작업에서 extension을 다른 파일로 분리하여 사용하다보니 private 접근제한자를 사용하지 않았었습니다.
이번에도 그럴거라 생각하고 무심결에 지정하지 않았네요! 지정할 필요가 있어보입니다.
통일성에 관해서는 아직 이 뷰가 연결될 수 있는 상황은 아니라 테스트겸으로 완성하다보니 미처 신경을 쓰지 못했었네요..